test/build/resolve-imports/expected-build/TEST_WMU/array-flatten.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Flatten an array indefinitely.
+ */
+function flatten(array) {
+    var result = [];
+    $flatten(array, result);
+    return result;
+}
+/**
+ * Internal flatten function recursively passes `result`.
+ */
+function $flatten(array, result) {
+    for (var i = 0; i < array.length; i++) {
+        var value = array[i];
+        if (Array.isArray(value)) {
+            $flatten(value, result);
+        }
+        else {
+            result.push(value);
+        }
+    }
+}
+
+export { flatten };
```

Comment:
I know that this file was originally excluded, but I added it back because it’s just 2 files to snapshot. It’s hard to say “ignore this folder” the way the tests are set up.